src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -82,7 +82,7 @@ private string GetRPCAuth()
 
         public void NotInIBD()
         {
-            ((InitialBlockDownloadStateMock) this.FullNode.NodeService<IInitialBlockDownloadState>()).SetIsInitialBlockDownload(false, DateTime.UtcNow.AddMinutes(5));
+            ((InitialBlockDownloadStateMock)this.FullNode.NodeService<IInitialBlockDownloadState>()).SetIsInitialBlockDownload(false, DateTime.UtcNow.AddMinutes(5));
```

Comment:
You are just bloating you PR with this kind of things

Reply:
Yeah please @fassadlr turn this auto-cleaning off, or do a PR that does this across the whole solution. ✨✨